src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs

```diff
@@ -62,10 +62,10 @@ await using (var stream = metadataFile.Open())
                     releaseId: releaseId,
                     stream: stream,
                     fileType: ReleaseFileTypes.Metadata,
-                    fileName: metadataFile.Name,
+                    fileName: metadataFile.Name.ToLower(),
                     contentType: "text/csv",
                     metaValues: GetMetaDataFileMetaValues(
-                        dataFileName: metadataFile.Name,
+                        dataFileName: dataFile.Name.ToLower(),
```

Comment:
No need to do this as the `GetMetaDataFileMetaValues` already lower cases the data file name

Reply:
reverted